pycbc/workflow/core.py

```diff
@@ -48,7 +49,7 @@ class ContentHandler(ligolw.LIGOLWContentHandler):
 
 # workflow should never be using the glue LIGOTimeGPS class, override this with
 # the nice SWIG-wrapped class in lal
-lal.LIGOTimeGPS = lalswig.LIGOTimeGPS
+gluelal.LIGOTimeGPS = lal.LIGOTimeGPS
```

Comment:
This should come from swig not glue. I think we wanted to move away from glue here, not back to it.

Reply:
@ahnitz, in the updated code `lal` really is `lal` (and not `glue.lal`), so this line change reflects the renamed module imports.

Reply:
Ah, right. Wouldn't it make more sense to change the following lines then to directly use lal.LIGOTimeGPS then?

Reply:
@ahnitz, I think the point here is that `glue.lal` has internal references to its own `LIGOTimeGPS` implementation, so you need to overwrite that if you want to have consistency amongst all things `LIGOTimeGPS`. @spxiwh can confirm.

Reply:
@duncanmmacleod  Indeed, I think you are right about that, and this line is still needed. I'm referring to the later lines, that use the LIGOTimeGPS object.

Reply:
This *should* be lal, not gluelal, as glue declares that glue.lal will soon not point to lal's LIGOTimeGPS

But glue will always be using the lal code here anyway (although it does have a fallback, that is actually broken and cannot be used). So there is no old-school glue LIGOTimeGPS with the current glue, in *any* circumstance.

Reply:
Resolved this by mostly using the lal CacheEntry and LIGOTimeGPS classes instead of ones provided by glue in this module ... I note that gluelal.Cache doesn't seem to have an equivalent in lal.utils, and this is needed to support "old-school" datafind usage (ie. when cache-files are needed).